lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+  attr_accessor :cells, :turn_count
+  #@cells = []
+def initialize
+  @cells = Array.new(9, " ")
+end
+
+def cells
+  @cells
```

Comment:
With `attr_accessor :cells`, this method is not needed. `attr_accessor` is a macro that automatically adds both the setter method and the getter method to your class for the selected attribute. In this case, both
```
def cells
  @cells
end
```
and
```
def cells=(cells)
  @cells = cells
end
```
have been included in your class through the `attr_accessor`.